src/main/resources/archetype-resources/silo/src/main/resources/application.yml

```diff
@@ -22,7 +22,7 @@ spring:
             maximum-pool-size: 3
             minimum-idle: 1
     application:
-        name: ${rootArtifactId}
+        name: ${rootArtifactId}-silo
```

Comment:
I am not sure about this. Shouldn't the rootArtifactId also be `product-silo`?

In the past we had the problem, that some projects names did follow the naming convention. So we had for example a database called `product-silo`, a message queue prefix of `product` and the spring application was simply called `product`. This problem appeared in all possible combination and made automating things harder.

No idea if that issue matters here.

Reply:
> Shouldn't the rootArtifactId also be product-silo?

I thought the same but in other places the same appending of `-silo` to the rootArtifactId  is done, f.ex. [here](https://github.com/rebuy-de/rebuy-silo-archetype/blob/master/src/main/resources/archetype-resources/deployment/k8s/branch/deployment.yaml#L6)

About the other part i have no idea, but i think this will keep it most consistent. Also without the `-silo` affix the silo does not start (not exactly sure what the error was but i think it was flyway related).

Reply:
Okay. Then it is fine for me.

Reply:
I think it could make sense to split "project-name" and the root-artifact-id. This way we avoid the confusion and can separate java world from the rest. Do you want to do that Martin?

Reply:
Yeah makes sense, will do. I would adjust it in the following way:
```
mvn archetype:generate \
...
    -DartifactId=tablarlift-simulation-silo \ (with silo suffix)
    -DprojectName=tablarlift-simulation \ (without silo suffix)
...
```

This would also allow to create a new silo without `-silo` in the name. What do you think?

Reply:
I think it should be the other way around, projectName should be same as the github repo name :)

Reply:
Ah yeah you are right.
```
mvn archetype:generate \
...
    -DartifactId=tablarlift-simulation \ (without silo suffix)
    -DprojectName=tablarlift-simulation-silo \ (with silo suffix)
...
```